app/frontend/src/pages/layout/Layout.module.css

```diff
@@ -13,9 +13,10 @@
 .headerContainer {
     display: flex;
     align-items: center;
-    justify-content: space-around;
-    margin-right: 12px;
-    margin-left: 12px;
+    justify-content: space-between;
```

Comment:
Since we adjusted the frontend a few weeks ago for PIM, maybe good to click through al pages/windows locally to verify the styling is still looking good. Maybe some style changes interfere with these changes from a few weeks ago.